modules/configuration/src/test/java/org/apache/ignite/internal/configuration/tree/NamedListNodeTest.java

```diff
@@ -347,4 +348,28 @@ public void errors() throws Exception {
         b.delete("X");
         b.delete("Y");
     }
+
+    /**
+     * Tests the {@link NamedListNode#update} method.
+     */
+    @Test
+    public void testUpdate() {
```

Comment:
Please add few to `org.apache.ignite.internal.configuration.tree.NamedListNodeTest#errors`

Reply:
I've added a separate test for the update errors, because I don't like the current approach where everything is cramped up in a single test method

Reply:
ok